setup.py

```diff
@@ -46,7 +46,7 @@ def read_version(rel_path):
     include_package_data=True,
     zip_safe=False,
     tests_require=['nose>=1.0', 'regex>=2020.4.4'],
-    install_requires=['regex>=2020.4.4'],
+    extra_requires=['regex>=2020.4.4'],
```

Comment:
```suggestion
    extras_requires=['regex>=2020.4.4'],
```

From a super quick google, does this need to be `extraS` ?